packages/discovery-react-components/src/components/DocumentPreview/components/PdfViewerWithHighlight/PdfViewerWithHighlight.tsx

```diff
@@ -165,37 +171,41 @@ function useHighlightState({
 /**
  * Hook to move PDF page depending on active highlight
  */
-function useMovePageToActiveHighlight(
+export function useMovePageToActiveHighlight(
   page: number,
   activeHighlightPages: number[],
+  activeIds: string[] | undefined,
```

Comment:
I think you can just make this optional instead of using `undefined`:
```suggestion
  activeIds?: string[],
```